_posts/2019-07-09-redhat-and-ibm.md

```diff
@@ -0,0 +1,40 @@
+---
+title: IBM acquisition of Red Hat
+author: Ina Panova
+tags:
+  - community 
+---
+
+Pulp community,
+
+Today marks a new day in the 26-year history of Red Hat. [IBM has finalized its acquisition of
+Red Hat](https://www.redhat.com/en/about/press-releases/
+ibm-closes-landmark-acquisition-red-hat-34-billion-defines-open-hybrid-cloud-future), which will
```

Comment:
I think the linebreak here needs to be removed. It's not displaying the link correctly for me.